packages/components/src/types/page.ts

```diff
@@ -138,6 +180,20 @@ export const ContentPagePageSchema = Type.Composite([
   imageSchemaObject,
 ])
 
+// NOTE: Previously, index page's header and content page's header
+// are identical but we are splitting them apart now.
+// This is the index page's header
+// should fully own the state of the collection/folder
+// but the content page header should not.
+// Doing a straight copy paste rather than `Type.Composite`
+// to avoid unexpected spillover of properties
+export const IndexPagePageSchema = Type.Composite([
+  Type.Object({
+    contentPageHeader: ContentPageHeaderSchema,
+  }),
+  imageSchemaObject,
+])
```

Comment:
technically this is not needed since the index page of a folder won't have the changes here (`tagCategories`) but i felt that it still makes sense to separate